src/Handlers/DatabaseHandler.php

```diff
@@ -142,7 +163,7 @@ private function hydrate()
         $rawValues = db_connect()->table($this->table)->get();
```

Comment:
I think our hydration system and caching system will need to change. If we have thousands of users in the system we don't want to load all of their individual settings into the system at once. I would recommend that `hydrate` could take the `$context`. If present it caches it locally under the context name, otherwise keeps another cache for non-context settings. 

```
private $settings = [
   'general' => [],
   'contexts' => []
];
```

Reply:
Good idea!